common/config/azure-pipelines/npm-publish.yaml

```diff
@@ -0,0 +1,15 @@
+resources:
+- repo: self
+variables:
+  NodeVersion: 10
+steps:
+- template: templates/build.yaml
+- script: 'node common\scripts\install-run-rush.js version --bump --version-policy BOGUS --target-branch $(Build.SourceBranchName)'
+  displayName: 'Rush Version'
+- task: npmAuthenticate@0

```

Comment:
This shouldn't be needed since you're using `--npm-auth-token` below with an environment variable. In my opinion the environment variable is safer, because `npmAuthenticate` writes a file to disk that may not get cleaned up properly.